docs/UserGuide.md

```diff
@@ -877,55 +877,77 @@ Format: `clear t/activity`
 
 <div style="page-break-after: always;"></div>
 
-## [**3.4. Finding entries: `find t/CATEGORY FIELD/[KEYWORDS]`**](#table-of-contents)
+## [**3.4. Finding entries:**](#table-of-contents)
 
-### [3.4.1 Finding a doctor through a keyword `find t/doctor FIELD/[KEYWORDS]`](#table-of-contents)
+### [3.4.1 Overview](#table-of-contents)
+The find feature allows you to search for entries based on user input by matching
+the input string with the user-specified fields.
+
+The model that is being searched must be specified too - doctor, patient or activity.
+
+For all find functions, the Keyword is case-insensitive for convenience (“dia” will match diabetic patients even if the user stored the patient's
+  condition as “Diabetes”)
+
+### [3.4.2 Finding a doctor through a keyword `find t/doctor FIELD/[KEYWORDS]`](#table-of-contents)
 
 Searches for doctors whose specified field contains one or more of the specified keywords as a substring.
-E.g. `find t/doctor n/Hans Bo` will return doctors whose names are `Hans Gruber`, or `Bo Yang`.
 
-Format: `find t/doctor FIELD/[KEYWORDS]`
+**Format**: `find t/doctor FIELD/[KEYWORDS]`
+
+**GoMedic** will display the matching doctors.  
 
 The possible parameters for FIELD are:
-* `n/NAME`: Matches the name field
-* `p/PHONE_NUMBER`: Matches the phone number field
-* `de/DEPARTMENT`: Matches the department field
 
-### [3.4.2 Finding a patient through a keyword `find t/patient FIELD/[KEYWORDS]`](#table-of-contents)
+Parameters    |  Explanation                                      |               
+--------------|---------------------------------------------------|
+`n/NAME`      | Matches the full name of the doctor                |
+`p/PHONE_NUMBER`| Matches the phone number of the doctor            | 
+`de/DEPARTMENT` | Matches the department of the doctor           |
```

Comment:
You might want to explain like the constraints are not checked, they only match the value (like you can put any phone number like p/abcde)

Reply:
Ok done